howto/troubleshoot/landing.md

```diff
@@ -1,11 +1,41 @@
 The topics in this guide describe some commonly encountered problems with Anbox Cloud and provide instructions for resolving them. If you encounter an issue with Anbox Cloud, check if any of the following scenarios help in resolving your issue.
 
+[note type="information" status="Note"]If the deployment is older than 3 months, you must upgrade Anbox Cloud to the latest version and see if the required fixes are already part of the upgrade. See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/how-to-upgrade-anbox-cloud/17750) for upgrade instructions.
+[/note]
+
 * [Troubleshoot issues with initial setup](https://discourse.ubuntu.com/t/troubleshoot-issues-with-initial-setup/35704)
 * [Troubleshoot container failures](https://discourse.ubuntu.com/t/troubleshoot-container-failures/35703)
 * [Troubleshoot issues with application creation](https://discourse.ubuntu.com/t/troubleshoot-issues-with-application-creation/35702)
 * [Troubleshoot issues with LXD clustering](https://discourse.ubuntu.com/t/troubleshoot-issues-with-lxd-clustering/35705)
 
-If you still need help, use the following instructions to collect troubleshooting information from the container and report an issue.
+If you still need help, use any of the following utilities to collect troubleshooting information and report an [issue](https://bugs.launchpad.net/anbox-cloud/+filebug).
+
+
+## Juju crashdump
+
+*Applies to: Anbox Cloud*
```

Comment:
Shouldn't we use something like `Applies to: Juju deployed Anbox Cloud`. "Anbox Cloud" refers to both, the Juju deployed one and the appliance.

Reply:
No, throughout the docs, we use Anbox Cloud for the regular variant and Anbox Cloud Appliance for the appliance variant. We have the variants explained in line with this in the overview topic (https://anbox-cloud.io/docs/exp/anbox-cloud) as well.

Reply:
Ok, lets go for consistency then. However I have a doubt that people will be able to understand this or read the explanation. Maybe we add a link here to the explanation?

Reply:
@morphis I have added a note before listing the troubleshooting utilities explaining that there could be variants and also a link to the explanation. Hope this helps.

Reply:
Thanks!